chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -101,7 +102,7 @@
     "provisioner": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 12,
+      "schema-revision": 13,
```

Comment:
Bump needed. Done in https://github.com/toabctl/barclamp-provisioner/commit/822fab6c191a4c559ebc2975a9a8f82403b57753